### PR TITLE
Create Jake Adams post

### PR DESCRIPTION
**Pull request by atribolet**
Content edited and reviewed. Waiting on a photo of Jake and a possible link to add in.

### CODE REVIEW

**Comment by atribolet** [comment]
Hey, @steveoh , this is ready for you!

File: _posts/_posts/2019-06-21-meet-jake-adams-new-agrc-staff.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: ' Meet Jake Adams - New AGRC Staff'
```
Comment (steveoh) [inline _posts/_posts/2019-06-21-meet-jake-adams-new-agrc-staff.md]: What keeps creating the space here? Can I fix that somehow?

```suggestion
title: 'Meet Jake Adams - New AGRC Staff'
```

**Comment by steveoh** [comment]
And you created this pr to your own repository again not to agrc

**Comment by atribolet** [comment]
Sorry :( K, what do I need to do to fix it? I tried!

**Comment by steveoh** [comment]
Close this pr. Then create a new one. When you go to the first screen you will be able to choose the base and it should be agrc master. https://help.github.com/en/articles/creating-a-pull-request#creating-the-pull-request

I tried to edit a current pr with rick the other day and it wouldn't let us change the base to a different repository.

**Pull request closed without merging**